packages/zwave-js/src/lib/commandclass/MultilevelSwitchCC.ts

```diff
@@ -705,19 +749,27 @@ export class MultilevelSwitchCCStartLevelChange extends MultilevelSwitchCC {
 	) {
 		super(driver, options);
 		if (gotDeserializationOptions(options)) {
-			// TODO: Deserialize payload
-			throw new ZWaveError(
-				`${this.constructor.name}: deserialization not implemented`,
-				ZWaveErrorCodes.Deserialization_NotImplemented,
-			);
+			validatePayload(this.payload.length >= 3);
+			const direction = (this.payload[0] & 0b0_1_0_00000) >>> 6;
+			const ignoreStartLevel = (this.payload[0] & 0b0_0_1_00000) >>> 5;
+			const startLevel = this.payload[1];
+			const duration = this.payload[2];
+
+			this.duration = Duration.parseReport(duration);
+			this.ignoreStartLevel = !!ignoreStartLevel;
+			this.startLevel = startLevel;
+			this.direction = direction ? "down" : "up";
+			this.eventType = MultilevelSwitchCommand.StartLevelChange;
 		} else {
 			this.duration = options.duration;
 			this.ignoreStartLevel = options.ignoreStartLevel;
 			this.startLevel = options.startLevel ?? 0;
 			this.direction = options.direction;
+			this.eventType = MultilevelSwitchCommand.StartLevelChange;
 		}
 	}
 
+	public eventType: MultilevelSwitchCommand;
```

Comment:
The class instance already identifies this command, no need for an extra variable. Just handle this like you handle the StopLevelChange command.

Reply:
Same goes for the other 2-3 lines where you reference this.